frontend/src/component/feature/FeatureView/FeatureOverview/FeatureLifecycle/LegacyFeatureLifecycleTooltip.test.tsx

```diff
@@ -0,0 +1,161 @@
+import { screen } from '@testing-library/react';
```

Comment:
I guess we have the same code in the FeautreLifecycleTooltip.test.tsx and this one will be soon removed so we keep git history for the new code

Reply:
Not 1:1 the same, but yes. I'll remove this file